docs-2.0/5.configurations-and-logs/1.configurations/3.graph-config.md

```diff
@@ -0,0 +1,75 @@
+# Graph Service configurations
+
+This document gives some introductions to Graph Service configurations. Graph Service configurations are stored in `nebula-graphd.conf.default` and `nebula-graphd.conf.production`. The configuration files are stored in `/usr/local/nebula/etc/` by default. If you have customized your Nebula Graph installation directory, the path to your configuration files is `$pwd/nebula/etc/`.
+
+* The `nebula-graphd.conf.default` file is the **default configuration file** for the Graph Service. We suggest that you use it for **debugging**. When you start Nebula Graph for the first time, Nebula Graph uses the default configuration files.
+* The `nebula-graphd.conf.production` file is the **production configuration file** for the Graph Service. We suggest that you use it for the **production environment**. To use it, you must remove the `.production` suffix.
+
```

Comment:
NOTE: If you want to change any parameter in the following, remember to add --local_conf=true at the top of this file.